backend/src/views/backend/page/personnel/index.ejs

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <%- include('../../layout/head.ejs') -%>
+  </head>
+  <body>
+    <%- include('../../layout/header.ejs') -%>
+
+    <main>
+      <div class="hero">
+        <h1><%= title %></h1>
+
+        <a class="button" href="/personnel/new">New Personnel</a>
```

Comment:
When this button is pressed, it prompts the screen/layout to allow the user to create a new personnel?

Reply:
Yep, just redirects to a different page

Reply:
There isn't any fancy JS stuff going on here, every screen is its own page :)